pay-api/src/pay_api/services/payment_calculations.py

```diff
@@ -0,0 +1,224 @@
+"""Service Helper to payment statement related calculations."""
```

Comment:
add this file because: src/pay_api/services/payment.py:1:0: C0302: Too many lines in module (1057/1000) (too-many-lines)